bin.src/addSeds.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import argparse
+import warnings
+import numpy as np
+from lsst.sims.movingObjects import Orbits
+from itertools import repeat
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Add SEDs to a series of orbits.")
+    parser.add_argument("--orbitFile", type=str, default=None,
```

Comment:
And yes, you would receive a complaint, I think.
If you paste in that line + the first few lines of one of your files, I can double-check.